tests/python/pants_test/pantsd/pantsd_integration_test.py

```diff
@@ -596,28 +596,23 @@ def test_dependencies_swap(self):
             )
             """
         )
```

Comment:
Refactors to this test aren't related. But I used it as my template for the real change, so I went and cleaned things up.